ui-v2/app/components/popover-select/index.js

```diff
@@ -4,6 +4,8 @@ import { inject as service } from '@ember/service';
 export default Component.extend({
   tagName: '',
   dom: service('dom'),
+  multiple: false,
```

Comment:
We default to single select for now (as was the previous functionality) but changing this to multiple will let you carry on selecting.